Updated to fix issue on mobile when OS goes outside of the view window

dskay3 opened the pull request:
OMNI-4180

mattgoldspink commented on ui/utilities/common-newport/_index.scss:
```diff
@@ -688,4 +688,14 @@ form {
+.via-nds .nds-form--multi-input__container .nds-tooltip_container .nds-nonfocused_control .nds-is-absolute .nds-popover.nds-popover_tooltip.nds-nubbin_bottom,
+.via-nds .nds-lookup .nds-tooltip_container .nds-nonfocused_control .nds-is-absolute .nds-popover.nds-popover_tooltip.nds-nubbin_bottom,
+.via-nds .nds-typeahead>div.nds-grid .nds-tooltip_container .nds-nonfocused_control .nds-is-absolute .nds-popover.nds-popover_tooltip.nds-nubbin_bottom {
+  left: calc(0% - 30px);
```
One other thing @dskay3 is that we should write `left` as `#{$left}` so that we can make it automatically flip when we compile the RTL stylesheets (Same goes for the right rule below).

dskay3 replied:
Gotcha, will do!

dskay3 commented:
@mattgoldspink, 
Updates should be in now. The OS Previewer was also updated to include the new class and updated tooltip class change. I've tested the changes in via_resources and they look good 👍
